src/utils.js

```diff
@@ -1,6 +1,12 @@
 
 import fetchJsonp from 'fetch-jsonp';
 
+export const specVersions={
+"0.1":"0.1",
+"0.2":"0.2",
+"0.3":"0.3",
+"draft":"draft",
```

Comment:
What is the motivation behind this object? If the keys are equal to the values, maybe it could be simply a list?

Reply:
@wetneb Yes, you are correct. If the keys are the same as the values, we can store it as a list. The primary motivation was to keep the versions in an object for easy reference throughout the code, ensuring we only need to update the value in one place if it changes.